.nais/dev-gcp.yml

```diff
@@ -12,7 +12,6 @@ metadata:
 spec:
   image: {{ image_dev_gcp }}
   ingresses:
-    - https://pensjon-selvbetjening-opptjening-frontend.dev.nav.no
```

Comment:
Brukes denne noe sted? Må det endres i PSELV for eksempel?

Reply:
Nei, den brukes ikke noe sted som jeg vet om.